users/views/identity.py

```diff
@@ -370,3 +370,10 @@ def form_valid(self, form):
         )
         self.request.session["identity_id"] = identity.id
         return redirect(identity.urls.view)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["user"] = self.request.user
+        if len(context["form"].fields["domain"].choices) == 0:
```

Comment:
This feels a bit ugly to read from the form field choices list. I'm not familiar enough with django formviews to know where to fetch domains so it's passed to both the form class init as well as have it in context.

Reply:
Yeah, it's not ideal, but not terrible - I'll merge it and then see if I can replace it with a more direct query.